src/transformers/pipelines/base.py

```diff
@@ -928,20 +943,49 @@ def save_pretrained(self, save_directory: str, safe_serialization: bool = True):
             # Save the pipeline custom code
             custom_object_save(self, save_directory)
 
-        self.model.save_pretrained(save_directory, safe_serialization=safe_serialization)
+        # TODO:
+        # depricate the safe_serialization parameter and use kwargs instead
+        # or update the save_pretrained to get all the parameters such as max_shard_size, ...
```

Comment:
TODO not finished here!

Reply:
since we are passing everything to the 
```diff
+ kwargs["safe_serialization"] = safe_serialization
+ self.model.save_pretrained(save_directory, **kwargs)
```
as kwargs i felt we should switch to a kwargs annotation
also yeh you are right, there is no need for deprecation or any changes, it already works perfectly as is, should i remove that comment ?

Reply:
Yep - we don't want to leave unnecessary TODOs in the codebase!

Reply:
removed the extra comment ✅